### PR TITLE
Added snackbar

### PR DESCRIPTION
**Pull request by shiv07tiwari**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
`

# Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings

### CODE REVIEW

File: Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContact.java [hunk 1]
```diff
@@ -163,9 +171,9 @@ public void onClick(View view) {
                 try {
                     startActivityForResult(qrDroid, ACTIVITY_CREATE);
                 } catch (ActivityNotFoundException activity) {
-                    Toast.makeText(ShareContact.this,
-                            "can't be generated. Need to download QR services",
-                            Toast.LENGTH_SHORT).show();
+                    Snackbar.make(Objects.requireNonNull(ShareContact.this).findViewById(android.R.id.content),
+                            R.string.cant_generate,
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContact.java]: Change cant_generate to snackbar_cant_generate_qrcode

File: Android/Android.iml [hunk 2]
```diff
@@ -8,12 +8,12 @@
       </configuration>
```
Comment (Swati4star) [inline Android/Android.iml]: Remove this file from PR.

File: Android/app/src/main/java/io/github/project_travel_mate/LocationService.java [hunk 3]
```diff
@@ -156,6 +156,7 @@ public void onLocationChanged(final Location loc) {
 
         public void onProviderDisabled(String provider) {
             Toast.makeText(getApplicationContext(), "Gps Disabled", Toast.LENGTH_SHORT).show();
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/LocationService.java]: Add this string also to strings.xml file

**Comment by shiv07tiwari** [comment]
Working on the changes.

File: Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContact.java (hunk 1 above)
Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/utilities/ShareContact.java]: change here also

**Comment by Swati4star** [comment]
Resolve merge conflicts

**Comment by Swati4star** [comment]
@shiv07tiwari Status?

**Comment by shiv07tiwari** [comment]
Trying to resolve the Conflicts :(

**Comment by Swati4star** [comment]
Closing this PR because of inactivity.

**Pull request closed without merging**